velox/functions/prestosql/aggregates/AverageAggregate.cpp

```diff
@@ -106,16 +107,16 @@ class AverageAggregate : public exec::Aggregate {
         if (decodedRaw_.isNullAt(i)) {
           return;
         }
-        updateNonNullValue(groups[i], decodedRaw_.valueAt<T>(i));
+        updateNonNullValue(groups[i], decodedRaw_.valueAt<TInput>(i));
```

Comment:
`updateNonNullValue(groups[i], decodedRaw_.valueAt<TInput>(i));` must be
`updateNonNullValue(groups[i], TAccumulator(decodedRaw_.valueAt<TInput>(i)));` since we changed the argument type of `updateNonNullValue`.
 Can you verify other invocations in a similar manner?